duckdb-vortex/src/vortex_scan.cpp

```diff
@@ -480,7 +483,11 @@ void RegisterScanFunction(DatabaseInstance &instance) {
 		auto global_state = make_uniq<ScanGlobalState>();
 
 		// TODO(joe): do this expansion gradually in the scan to avoid a slower start.
-		global_state->expanded_files = bind.file_list->GetAllFiles();
+		auto file_infos = bind.file_list->GetAllFiles();
+		global_state->expanded_files.reserve(file_infos.size());
+		for (const auto &file_info : file_infos) {
+			global_state->expanded_files.push_back(file_info.path);
+		}
```

Comment:
For me understanding, why do this?

Reply:
We can't simply assign anymore as we need to access `.path`. Alternatively, we change the vector type for `expanded_files`.